addons/breathing/functions/fnc_getETCo2.sqf

```diff
@@ -0,0 +1,78 @@
+#include "..\script_component.hpp"
+/*
+ * Author: apo_tle
+ * Calculate patient's ETCo2 level based on status of their vitals.
+ *
+ * Arguments:
+ * 0: Patient <OBJECT>
+ *
+ * Return Value:
+ * ETCo2 level and units <STR>: "40 mmHg"
+ *
+ * Example:
+ * [player] call kat_breathing_fnc_getETCo2;
+ *
+ * Public: No
+ */
+
+params ["_patient"];
+
+if !(alive _patient) exitWith {0};
+
+private _pr = GET_HEART_RATE(_patient);
+private _bloodVolume = GET_BLOOD_VOLUME(_patient);
+private _lostBlood = 6.0 - _bloodVolume;
+private _pH = _patient getVariable [QEGVAR(pharma,pH),1500];
+private _phDiff = 1500 - _pH;
+
+private _ptxTarget = _patient getVariable [QGVAR(pneumothorax), 0]; // more deteriorated closer to 4, at 4 likely to become advanced
+private _hasHtx = _patient getVariable [QGVAR(hemopneumothorax), false];
+private _hasTptx = _patient getVariable [QGVAR(tensopneumothorax), false];
+
+private _airwayObstructed = _patient getVariable [QEGVAR(airway,obstruction), false] && !(_patient getVariable [QEGVAR(airway,overstretch), false]);
+private _airwayOccluded = _patient getVariable [QEGVAR(airway,occluded), false];
+
+private _phOffset = [0,(floor (_phDiff / 250))] select (EGVAR(pharma,kidneyAction)); // increase etco2 by 1 per 250 points of pH lost
```

Comment:
Will break if pharma is not loaded

Reply:
Would an if statement checking for pharma being enabled fix this or would it still break?